### DISCUSSION TITLE
Standup (Monday, May 16, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#63] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - Start developing again 
 - Talk about authentication with @CarlosCRG19 and @BrandonMagana 

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19 for deploying the app

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Deployment of app
 - Defined environment variables (#33)

**What are your priorities for the next 24 hours?**:
 - Define new tasks with the team
 - Review authentication options

**Blockers**:
 - Pending planning

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Didn't do anything

**What are your priorities for the next 24 hours?**:
 -  Return back to coding 
 -  Reorganize activities and tasks with @antonieto 

**Blockers**:
 - None

**Shoutouts**:
 - @CarlosCRG19  for deploying the app ILY, @paotg123 for coming back

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Zzz

**What are your priorities for the next 24 hours?**:
 - Know what's been going on
 - Define activities to do

**Blockers**:
 - Nothing anymore

**Shoutouts**:
 - @paotg123 for keeping up with us, and @CarlosCRG19 for deploying

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - None

**What are your priorities for the next 24 hours?**:
 - Define tasks

**Blockers**:
 -

**Shoutouts**:
 - @CarlosCRG19 for deploying
